pr_agent/tools/pr_reviewer.py

```diff
@@ -93,8 +94,12 @@ async def run(self) -> None:
         """
         Review the pull request and generate feedback.
         """
-        logging.info('Reviewing PR...')
-    
+        if self.is_auto and not get_settings().pr_reviewer.automatic_review:
+            logging.info(f'Automatic review is disabled {self.pr_url}')
```

Comment:
**Suggestion:** Consider adding a log message when automatic review is disabled. This will help in understanding the program's execution flow.
```suggestion
        if self.is_auto and not get_settings().pr_reviewer.automatic_review:
            logging.info(f'Automatic review is disabled for PR: {self.pr_url}')
            return None
```